.github/actions/run-e2e/action.yaml

```diff
@@ -162,6 +162,7 @@ runs:
       sudo crictl images > crictl.images
       sudo podman stats -a --no-stream --no-reset > podman.stats
       journalctl -u kubelet > kubelet.log
+      journalctl -u kube-proxy > kube-proxy.log
```

Comment:
Aren't we already collecting it from a mirroring Pod?

Reply:
what's `a mirroring Pod`?

Reply:
you are right, we already have the logs from the static pod. I'll drop this one.

Reply:
dropped

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/ it's actually a mirror* Pod, but anyway a reflection of a static Pod in apiserver